include/nighthawk/client/factories.h

```diff
@@ -24,19 +24,44 @@ namespace Client {
 class BenchmarkClientFactory {
 public:
   virtual ~BenchmarkClientFactory() = default;
+
+  /**
+   * Constructs a BenchmarkClient
+   *
+   * @param api reference to the Api object.
+   * @param dispatcher supplies the owning thread's dispatcher.
+   * @param scope stats scope for any stats tracked by the benchmark client.
+   * @param cluster_manager Cluster manager preconfigured with our target cluster.
+   * @param http_tracer http tracer
```

Comment:
(nit) Do we have anything more interesting to say about this, maybe what it is used for pr what it means in the context of this function? The current comment is kind of redundant.

Reply:
yeah, that wasn't a very helpful doc comment :-)
done in 28487f0